src/minlog.js

```diff
@@ -1,129 +1,36 @@
 import _ from 'lodash';
-
```

Comment:
please do not submit PRs with content from others, when there's no dependency between them